migration/migrator/migrations/course/20231005130620_forum_attachment_names.py

```diff
@@ -0,0 +1,112 @@
+"""Migration for a given Submitty course database."""
+import os
+#import pdb; pdb.set_trace()
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("""
```

Comment:
It might be better to have only one database.execute() call in case something fails along the way. This might mean storing all of the sql string in one variable and then calling database.execute() on the variable. Also, the formatting for this sql string is kinda hard to read, is there possibly a more legible way of writing (if this is just simply the correct way of writing the string let me know, I've never written a string this complicated before)

Reply:
Just changed the string formatting. This is how my friend who was helping me told me to format queries but I agree with you this looks better to my eyes.